R/recall_by_manufacturer.R

```diff
@@ -0,0 +1,37 @@
+#ua <- user_agent("http://github.com/WrathSmith/caRecall")
+#ua
+#' recall_by_manufacturer
+#'
+#' @param manufacturer: A car manufacturer name (ie. Honda, Nissan, etc.)
+#' limit: Number of records to return (default is 25)
+#' page: the serach page result to return (default is 1)
+#'
+#'
+#' @return text output
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' get_recall_by_number(1977044)
+#' }
+recall_manufacturer <- function(manufacturer, limit=25, page=1) {
```

Comment:
It is generally accepted to use verbs in function names.

From our style guide.
As well as following the general advice for object names, strive to use verbs for function names:
How strictly do we want to interpret strive?

Reply:
I think as long as the function name makes sense and isn't too long, it's ok but if given the choice, go for the verb option (but if using the verb option makes it seem odd, then go with the noun). Maybe just think about what makes sense if you were a new user using the package. Could always quickly look at other API wrapper packages and check what typical conventions are used. There are a lot of functions in the tidyverse that aren't verbs so seems like a bit of a loose direction aka strive lol.

Reply:
get_recall_manufacturer is a bit long. get_manufacturer or get_recall_man I would be fine with.

Reply:
Either of your suggestions are good.